### ISSUE TITLE
ui-bog:low - user experience test 

### ISSUE THREAD

**Issue by MJdiop** [argoapp-live/argo-bounty-issue#9]
Wouldn't it be better for the user experience to have a button to close the menu on the mobile version  

![mobile-test](https://user-images.githubusercontent.com/42707049/125751129-15475ef3-ade2-4faf-89ec-9cc493c9620f.jpeg)

**Comment by wanisramdani** [comment]
Not really needed you can click on the right side (where the red rectangle) and the sidebar will close.

![Screenshot_20](https://user-images.githubusercontent.com/34271234/125810378-1b1c81d0-d6e3-4780-ba55-50db20b145c5.png)

**Comment by MJdiop** [comment]
yes indeed, I think it would be more natural for a user to have the close  button(X)

**Comment by izrake** [comment]
@MJdiop Agree, but our UX was designed like this. For sure, we will take it into consideration.
